files/default/ChefClientUpdater.ps1

```diff
@@ -41,20 +41,6 @@ function Log-Event {
   Write-Host $Message
 }
 
-# ----------------------------------------------------
```

Comment:
Not keen to remove this as I plan to add a facility for the updater to pull config from a central location which would allow switching to another version if the version we just installed is problematic (eg. If deprecations cause a CCR failure). The frequency probably needs to be configurable though.